modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java

```diff
@@ -601,6 +641,7 @@ private boolean _hasAddKBFolderPermission() throws PortalException {
 			KBActionKeys.ADD_KB_FOLDER);
 	}
 
+	private String _displayStyle;
```

Comment:
Do you know any reason why the attribute `_displayStyle` cannot be `final` and be initialized in the constructor? Maybe @AliciaGarciaGarcia has more information about this?

Reply:
I don't know... in some places I saw it with `final` and in others not... also sometimes is passed as parameter in the constructor... I had a lot of doubts here and that is why I'm requesting your review. Also, this way does not work good, because whenever you reload, it shows the `list` view as default instead of the last view (if it was table). So I guess I'm not obtaining this variable as I should...

Reply:
IMO the generation of the `_displayStyle` will need a method, and I do not think a good idea to initialize this variable with a method on the constructor.
You should add to the constructor the new param if you want the variable be final.

I do not think it is a good idea.

Reply:
Would you guys be so kind to add to this pull the changes you request? I don't think I really understand what needs to be done, and this is pure backend. Thanks!

Reply:
Yes! I'm on it ;-)

Reply:
Thank you very much @marcogalluzzi